pkg/sysinfo/cgroup2_linux.go

```diff
@@ -66,6 +69,29 @@ func newV2(quiet bool, opts *opts) *SysInfo {
 	return sysInfo
 }
 
+func getSwapLimit() bool {
```

Comment:
@AkihiroSuda I hope this one is better now. I have tested the logic over several distros and it seems to be working. Let me know what you think.